vms/rpcchainvm/vmproto/vm.proto

```diff
@@ -97,6 +97,12 @@ message BlockRejectRequest {
     bytes id = 1;
 }
 
+message SaveBlockRequest {
+    bytes bytes = 1; // The byte representation of the block to be saved
+}
+
+message SaveBlockResponse {}
+
```

Comment:
Can we move this down so that `BlockRejectRequest` and `BlockRejectResponse` are next to each other like all the other requests?